api/app/controllers/api/v1/instructors_controller.rb

```diff
@@ -1,60 +1,32 @@
 # frozen_string_literal: true
 
 module Api::V1
-  # Controller for Instructors
-  class InstructorsController < ApplicationController
-    before_action :set_instructor, only: %i[show update destroy]
-
-    # GET /instructors
-    def index
-      @instructors = Instructor.order(:id)
-
-      render json: @instructors
-    end
-
-    # GET /instructors/1
-    def show
-      render json: @instructor
-    end
-
-    # POST /instructors
-    def create
-      @instructor = Instructor.new(instructor_params)
-
-      if @instructor.save
-        render json: @instructor, status: :created
-      else
-        render json: @instructor.errors, status: :unprocessable_entity
-      end
-    end
-
-    # PATCH/PUT /instructors/1
-    def update
-      if @instructor.update(instructor_params)
-        render json: @instructor
-      else
-        render json: @instructor.errors, status: :unprocessable_entity
-      end
-    end
-
-    # DELETE /instructors/1
-    def destroy
-      if @instructor.destroy
-        head :no_content, status: :ok
-      else
-        render json: @position.errors, status: :unprocessable_entity
-      end
-    end
+    # Controller for Instructors
+    class InstructorsController < ApplicationController
+
+        # GET /instructors
+        def index
+            render json: { status: 'success', message: '', payload: Instructor.order(:id) }
+        end
+
+        # POST /instructors
+        def create
+            instructor = Instructor.new(instructor_params)
+            if instructor.save
```

Comment:
What is this?

Reply:
Which line are you referring to?

Reply:
`if instructor.save`

Reply:
This checks if the `Instructor` created passes all the validation for the model `Instructor`.

Reply:
`.save` seems like a really awkward name for `.passes_validation`...

Reply:
Ah, I understand. `instructor.save` does the action of saving an instructor returning `true` if the save was successful. Can you add a comment to this effect for new rails people?

Reply:
Okay. I'll do that.